src/storage/heap_file.c

```diff
@@ -5235,7 +5235,6 @@ heap_manager_finalize (void)
  *   hfid(in/out): Object heap file identifier.
  *                 All fields in the identifier are set, except the volume
  *                 identifier which should have already been set by the caller.
- *   exp_npgs(in): Expected number of pages
```

Comment:
How about updating the return type together? The description said it it HFID * but it is int.

Reply:
`estimated from the number of expected pages.`

The comment should be changed together. However it is hard to you at this moment. So, think about it and share your opinion.

Reply:
I think it would be better to deal with it all at once. I will reply once i have analyzed this function.